Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -712,8 +712,16 @@ protected virtual void SetupEvents()
         {
             for (int i = 0; i < InteractableEvents.Count; i++)
             {
-                InteractableEvents[i].Receiver = InteractableEvent.CreateReceiver(InteractableEvents[i]);
-                InteractableEvents[i].Receiver.Host = this;
+                var receiver = InteractableEvent.CreateReceiver(InteractableEvents[i]);
+                if (receiver != null)
+                {
+                    InteractableEvents[i].Receiver = receiver;
+                    InteractableEvents[i].Receiver.Host = this;
+                }
+                else
+                {
+                    Debug.LogWarning($"Empty event receiver found on {gameObject.name}, you may want to re-create this asset." );
```

Comment:
If I was a user that saw, this how would I know how to recreate the asset?

Reply:
Great question. I think they just need to set up the profiles and events again, which is not great. I want to improve this message once we understand this bug better, but for now I think not throwing errors will be good